index.js

```diff
@@ -71,10 +80,23 @@ function getName(/* code here */) {
  *         and returns a string like `Hello, my name is {name}`.
  *         where `{name}` is the name passed into `makeSmartPerson`.
 */
-function makeSmartPerson(/* code here */) {
-  /* code here */
+function makeSmartPerson(name) {
+  const obj = {
+    name: name,
+    sum: function(a, b){
+      return a + b;
+    },
+    speak: function() {
+      return `Hello, my name is ${this.name}` ;
+    }
+  };
+  return obj;
 }
```

Comment:
line 86 to 91: Here is a good place to use arrow function because they are anonymous functions and only have one statement.